modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -127,6 +127,7 @@ inline int toCV(const ONNXTensorElementDataType prec) {
     case ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT8: return CV_8U;
     case ONNX_TENSOR_ELEMENT_DATA_TYPE_FLOAT: return CV_32F;
     case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT32: return CV_32S;
+    case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64: return -1;
```

Comment:
As option `ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64` can be mapped `CV_32S` and setup the flag `is_conversion`
to prevent handling `-1` in every place. 

What do you think ? Let's ask someone else for example

Reply:
Function returns tuple<type, is_converted>. For example for ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT8 -> make_tuple(CV_U8, false).